wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -2301,6 +2302,10 @@
 					onDrag = jQuery.noop;
 				}
 
+				if (typeof(settings) !== "undefined" && settings.stopDragOnCssSelector) {
```

Comment:
According to the code `settings` should be `object`
Maybe it would be better to change this line to be
`typeof(settings) === 'object'` ?

Reply:
True. I'll change it to === 'object'. Will check that it still works and push.